src/mount/mount-tool.c

```diff
@@ -296,6 +296,31 @@ static int parse_argv(int argc, char *argv[]) {
                         log_error("Listing devices only supported locally.");
                         return -EOPNOTSUPP;
                 }
+        } else if (arg_action == ACTION_UMOUNT) {
+                if (optind >= argc) {
+                        log_error("At least one argument required.");
+                        return -EINVAL;
+                }
+
+                if (argc > optind+1) {
+                        log_error("At most one argument required.");
+                        return -EINVAL;
+                }
+
+                if (is_device_path(argv[optind])) {
```

Comment:
I need this check to determine if it is a dev path or a mount point. This is needed to set the arg_discover which is only for dev paths.